q2cli/tools.py

```diff
@@ -19,21 +19,23 @@ def tools():
 
 
 @tools.command(name='export',
-               short_help='Export data from a QIIME 2 Artifact'
+               short_help='Export data from a QIIME 2 Artifact '
                'or a Visualization',
-               help='Exporting extracts (and optionally transforms) data'
-               'stored inside an Artifact or Visualization. Note that'
+               help='Exporting extracts (and optionally transforms) data '
+               'stored inside an Artifact or Visualization. Note that '
                'Visualizations cannot be transformed with --output-format'
                )
 @q2cli.option('--input-path', required=True,
               type=click.Path(exists=True, file_okay=True,
-                              dir_okay=False, readable=True))
+                              dir_okay=False, readable=True),
+              help='Path to file that should be exported')
 @q2cli.option('--output-path', required=True,
               type=click.Path(exists=False, file_okay=True, dir_okay=True,
                               writable=True),
-              help='Directory where data should be exported to')
+              help='Path to file or directory where '
+              'data should be exported to')
 @q2cli.option('--output-format', required=False,
-              help='Format which the data should be exported as. '
+              help='Format which the data should be exported as. \n'
```

Comment:
--output-format was being broken down into two words when shown on the command line. I added a new line to prevent this!

Reply:
Good catch! How about instead we say `Format which [...]. This option cannot be used with Visualizations.`?

Reply:
I like that! I will change it now ⚙️